src/kv/RocksDBStore.cc

```diff
@@ -3012,7 +3011,13 @@ KeyValueDB::Iterator RocksDBStore::get_iterator(const std::string& prefix, Itera
         std::move(bounds));
     }
   } else {
-    return KeyValueDB::get_iterator(prefix, opts);
+    // use wholespace engine if no cfs are configured
+    // or use default cf otherwise as there is no
+    // matching cf for the specified prefix.
+    auto w_it = cf_handles.size() == 0 || prefix.empty() ?
```

Comment:
`prefix` cannot be empty for Iterator interface.
There is no way to inspect prefix associated with the key.
I guess that it is an assert condition.

Reply:
will fix in a follow-up PR as the current one is under QA atm and the issue is rather minor IMO